test/WebSites/MvcTagHelpersWebSite/Controllers/MvcTagHelper_HomeController.cs

```diff
@@ -162,6 +162,11 @@ public IActionResult Link()
             return View();
         }
 
+        public IActionResult Image()
```

Comment:
Need another test in [`MvcTagHelpersTest`](https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTagHelpersTest.cs).  This addition isn't confirmed otherwise.